app/src/main/java/com/fpvout/digiview/UsbMaskConnection.java

```diff
@@ -59,4 +77,16 @@ public void stop() {
     public boolean isReady() {
         return ready;
     }
+
+    public void setUsbDevice(UsbManager usbManager, UsbDevice d, DVR _dvr) {
+        dvr = _dvr;
+        usbConnection = usbManager.openDevice(d);
+        usbInterface = d.getInterface(3);
+
+        usbConnection.claimInterface(usbInterface, true);
+
+        mOutputStream = new AndroidUSBOutputStream(usbInterface.getEndpoint(0), usbConnection);
+        mInputStream = new AndroidUSBInputStream(usbInterface.getEndpoint(1), usbInterface.getEndpoint(0), usbConnection, dvr);
```

Comment:
we probably don't want to inject directly dvr into the inputStream (feels really akward) but more the other way around.
what I think would look good, is to set a method on UsbMaskConnection to register a rawStreamListener (would also allow output stream through RTMP to work i.e)

My approach for stream consuming model is :
- to set a master stream consumer that is responsible for consuming and imposing the timing on the USB stream input. (ie the h264Reader on Exoplayer through its DataSource implementation)
- to set "slave" raw stream listeners that just receive copies of data as they arrived 

Could work on something and PR it to you.